variables.tf

```diff
@@ -28,7 +28,7 @@ variable "mysql_version" {
 
 variable "region" {
   type        = string
-  description = "The region where you want to deploy your instance."
+  description = "The region to provision all resources in. [Learn more](https://terraform-ibm-modules.github.io/documentation/#/region) about how to select different regions for different services."
```

Comment:
this description is for DAs that have addon only. check in https://github.ibm.com/GoldenEye/issues/issues/13599  Sync with Redis please